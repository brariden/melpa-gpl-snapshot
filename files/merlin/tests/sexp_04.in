#!/bin/bash
. functions.sh || exit 1

reset sexp_04.ml
package sexplib
tell <<END
module Test_sexp = struct
  exception E of int with sexp
end
END
cmd errors
cmd dump env
