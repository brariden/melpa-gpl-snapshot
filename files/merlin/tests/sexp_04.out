["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"result":true}]
["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"cursor":{"line":4,"col":0},"marker":false}]
["return",{"cursor":{"line":4,"col":0},"marker":false}]
["return",[]]
["return",[["File \"sexp_04.ml\", line 2, characters 12-21","module Test_sexp : sig exception E of int end"]]]
